grpc-testtool/proto/merkle/merkle.proto

```diff
@@ -7,16 +7,22 @@ import "google/protobuf/empty.proto";
 // Methods on this service return status code NOT_FOUND if a requested
 // view, iterator or root hash is not found.
 service Merkle {
+  // --- Proposals ---
   rpc NewProposal(NewProposalRequest) returns (NewProposalResponse);
   rpc ProposalCommit(ProposalCommitRequest) returns (google.protobuf.Empty);
   
+  // --- Views ---
   rpc NewView(NewViewRequest) returns (NewViewResponse);
+
+  // --- Reads ---
   // The methods below may be called with an ID that corresponds to either a (committable) proposal
   // or (non-committable) historical view.
   rpc ViewHas(ViewHasRequest) returns (ViewHasResponse);
   rpc ViewGet(ViewGetRequest) returns (ViewGetResponse);
+
+  // --- Iterators ---
   rpc ViewNewIteratorWithStartAndPrefix(ViewNewIteratorWithStartAndPrefixRequest) returns (ViewNewIteratorWithStartAndPrefixResponse);
-  // Returns status code NOT_FOUND when the iterator is done.
+  // Returns status code OUT_OF_RANGE when the iterator is done
```

Comment:
No, the implementations are being developed. This change is because OUT_OF_RANGE is the documented correct error when you've reached the end of the set. From https://grpc.github.io/grpc/core/md_doc_statuscodes.html (emphasis mine):
```
The operation was attempted past the valid range. E.g., seeking or reading past end-of-file. Unlike INVALID_ARGUMENT, this error indicates a problem that may be fixed if the system state changes. For example, a 32-bit file system will generate INVALID_ARGUMENT if asked to read at an offset that is not in the range [0,2^32-1], but it will generate OUT_OF_RANGE if asked to read from an offset past the current file size. There is a fair bit of overlap between FAILED_PRECONDITION and OUT_OF_RANGE. We recommend using OUT_OF_RANGE (the more specific error) <b>when it applies so that callers who are iterating through a space</b> can easily look for an OUT_OF_RANGE error to detect when they are done.
```